docs/websocket/feedback.md

```diff
@@ -123,6 +159,16 @@ id   | `string` | **Required** The id of the feedback to pause.
 
 A paused feedback can be resumed using this RPC. After this the audio stream is
 again processed by the server.
+Currently **it is required to re-send the audio header when resuming
+feedback**. Failing to do so will fail the feedback as the server can't
+recognise the audio format without a header.
+
+!!! note
+    It appears to be valid to have a wave file with multiple headers.
```

Comment:
Did we test that? We could (re)generate the wave file without the extra headers btw.

Reply:
It seems to work in the tests and I read something about this in a description of the wave format.

Reply:
👍